source/usage-examples/bulkWrite.txt

```diff
@@ -41,11 +41,16 @@ documentation for full details.
   <Collection.html#~bulkWriteOpCallback>` object containing the
   result of the operation.
 
-``bulkWrite()`` returns a ``BulkWriteResult`` object
-that includes the number of updated or inserted documents, the ``_id``
-of each document, and any ``writeError`` or ``writeConcernErrors``.
-Refer to the documentation for :node-api:`BulkWriteResult
-</BulkWriteResult>` for more information.
+If you omit the callback parameter, ``bulkWrite()`` returns a
+:mdn:`Promise <Web/JavaScript/Reference/Global_Objects/Promise>` that
+resolves to a ``BulkWriteResult`` object that contains:
```

Comment:
I think the prior approach of linking to the API is likely more future-proof, documenting the properties of types should be left to API documentation IMO